<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>801</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Модель системы линз</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="objectParameters" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="objectXField" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>381</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="objectXFieldLabel">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>координата x</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="objectXLayout">
        <item>
         <widget class="QLineEdit" name="objectXInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="objectXInputLabel">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>11</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="objectParametersLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Параметры объекта</string>
     </property>
    </widget>
    <widget class="QWidget" name="objectSizeField" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>381</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="objectSizeFieldLabel">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>размер</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="objectSizeLayout">
        <item>
         <widget class="QLineEdit" name="objectSizeInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="objectSizeInputLabel">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>11</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="objectOrientationSelect" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>381</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="objectOrientationSelectLabel">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ориентация</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="objectOrientationLayout">
        <item>
         <widget class="QRadioButton" name="objectOrientationRBDown">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>вниз</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="objectOrientationRBUp">
          <property name="text">
           <string>вверх</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>-10</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="lensesParameters" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>400</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="lensesSelectField" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>381</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="lensesSelectFieldLabel">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>выбор линзы</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lensesSelectLayout">
        <item>
         <widget class="QComboBox" name="lensesSelectComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="maxVisibleItems">
           <number>5</number>
          </property>
          <property name="minimumContentsLength">
           <number>5</number>
          </property>
          <property name="placeholderText">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="lensesSelectCheckBox">
          <property name="text">
           <string>добавить</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="lensesParametersLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Параметры системы линз</string>
     </property>
    </widget>
    <widget class="QWidget" name="lensesXField" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>381</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLabel" name="lensesXFieldLabel">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>координата x</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lensesXFieldLayout">
        <item>
         <widget class="QLineEdit" name="lensesXInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lensesXInputLabel">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>11</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="lensesSettingChoice" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>381</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QRadioButton" name="lensesSettingRBFocalLen">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>фокусное расстояние</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lensesSettingFocalLenLayout">
        <item>
         <widget class="QLineEdit" name="lensesSettingFocalLenInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lensesSettingFocalLenInputLabel">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>(&lt;0 для рассеивающей)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="lensesSettingRBRadius">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>радиусы кривизны</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lensesSettingRadiusLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="lensesSettingR1Field" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QLabel" name="lensesSettingR1InputLabel">
             <property name="text">
              <string>R1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lensesSettingR1Input">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>14</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="lensesSettingR2Field" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLabel" name="lensesSettingR2InputLabel">
             <property name="text">
              <string>R2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lensesSettingR2Input">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>14</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="imageStats" native="true">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>509</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QLabel" name="imageStatsLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Изображение</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageStatsXLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>координата x:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageStatsSizeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>размер:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageStatsOrientationLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ориентация:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageStatsOrientationOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>вверх</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageStatsSizeOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>60.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageStatsXOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageStatsTypeLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>тип:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageStatsTypeOutput">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>РЕАЛЬНОЕ</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="menu" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>510</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QLabel" name="menuLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Меню</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menuCalculateButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menuSaveButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menuSaveAsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сохранить как...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menuOpenButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
    <widget class="QLabel" name="menuSaveFileLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Файл сохранения:</string>
     </property>
    </widget>
    <widget class="QLabel" name="menuSaveFileNameLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>save1.txt</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>510</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
